integrations/to.etc.domui.selenium/src/main/java/to/etc/domui/webdriver/core/base/Either.java

```diff
@@ -0,0 +1,70 @@
+package to.etc.domui.webdriver.core.base;
+
+import org.eclipse.jdt.annotation.Nullable;
+import to.etc.function.ConsumerEx;
+
+public class Either<RIGHT, WRONG> {
```

Comment:
Hate this. The name is definitely wrong; either just means one or the other, and implies nothing about right and wrong.
Looking at its usage I much, much prefer just erroring out in normal code.